pkg/sharing/replicator.go

```diff
@@ -553,38 +539,22 @@ func (s *Sharing) sendBulkDocs(inst *instance.Instance, m *Member, creds *Creden
 			"Authorization": "Bearer " + creds.AccessToken.AccessToken,
 		},
 		Body: bytes.NewReader(body),
-	})
+	}
+	res, err := request.Req(opts)
 	if err != nil {
 		return err
 	}
+	if res.StatusCode/100 == 5 {
+		return ErrInternalServerError
+	}
 	if res.StatusCode/100 == 4 {
 		res.Body.Close()
-		if err = creds.Refresh(inst, s, m); err != nil {
-			return err
-		}
-		res, err = request.Req(&request.Options{
-			Method: http.MethodPost,
-			Scheme: u.Scheme,
-			Domain: u.Host,
-			Path:   "/sharings/" + s.SID + "/_bulk_docs",
-			Headers: request.Headers{
-				"Accept":        "application/json",
-				"Content-Type":  "application/json",
-				"Authorization": "Bearer " + creds.AccessToken.AccessToken,
-			},
-			Body: bytes.NewReader(body),
-		})
-		if err != nil {
+		if res, err = RefreshToken(inst, s, m, creds, opts, body); err != nil {
```

Comment:
I think that we should close `res.Body` if err is `ErrInternalServerError` or `ErrClientError`